nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -311,6 +456,134 @@ def enable_pad_mask(self, on=True):
         self.use_pad_mask = on
         return mask
 
+    def setup_streaming_params(
+        self,
+        chunk_size=None,
+        shift_size=None,
+        cache_drop_size=None,
+        pre_encode_cache_size=None,
+        valid_out_len=None,
+        drop_extra_pre_encoded=None,
+    ):
+        MAX_LOOK_AHEAD = 10000
```

Comment:
Dont hardcode ? Or is this never going to change?

Reply:
made it an input.